api/modules/users/views.py

```diff
@@ -237,3 +238,31 @@ def remove_user_status(request):
         return Response(str(e), status=status.HTTP_400_BAD_REQUEST)
 
     return Response(status=status.HTTP_200_OK)
+
+
+@api_view(['GET'])
+def delete_profile(request):
+    """
+    Remove user profile
+    :param request:
+    :return: 404 if profile does not exist
+    :return: 200 successful
+    """
+    try:
+        user_profile = request.user.profile
+        if user_profile:
+            user_profile.delete()
+        else:
+            error_message = 'Profile does not exist'
+            return Response(error_message, status=status.HTTP_404_NOT_FOUND)
```

Comment:
If there is no `Profile`, then you do not need to send an error message, you can simply continue to delete trips